ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/class-map.j2

```diff
@@ -0,0 +1,16 @@
+{# class-map #}
+{% if class_maps is arista.avd.defined %}
+{%     if class_maps.qos is arista.avd.defined %}
```

Comment:
the natural_sort_filter now also test if the value is defined, so you can remove line 3 `{%     if class_maps.qos is arista.avd.defined %}` is not required!

Reply:
Got it! Fixed